components/mpas-ocean/src/mode_init/mpas_ocn_init_global_ocean.F

```diff
@@ -1204,28 +1018,25 @@ subroutine ocn_init_setup_global_ocean_interpolate_land_ice_topography(domain, i
                'indicates that this field is missing in the initial condition file (e.g. ' // &
                'because it is meant for an older E3SM version).', MPAS_LOG_CRIT)
           endif
-          ssh(:) = 0.0_RKIND
           landIceFraction(:) = 0.0_RKIND
-          modifyLandIcePressureMask(:) = 0
+          landIceDraft(:) = 0.0_RKIND
           landIcePressure(:) = 0.0_RKIND
+          modifyLandIcePressureMask(:) = 0
```

Comment:
It would be nice if this were called `sshAdjustmentMask` since we're not always modifying `landIcePressure`, but it's up to you if you want to bother.

Reply:
Sure, this will require changes outside of `global_ocean` but I'm okay with that.

Reply:
I have done this both here and in https://github.com/MPAS-Dev/compass/pull/813